spec/features/two_factor_authentication/multiple_mfa_sign_up_spec.rb

```diff
@@ -268,7 +268,10 @@
     end
 
     it 'regenerates backup codes path if a user clicks that they need new backup codes' do
-      click_link strip_tags(t('two_factor_authentication.backup_codes.new_backup_codes_html'))
+      find(
+        'a',
+        text: t('two_factor_authentication.backup_codes.new_backup_codes_html').gsub('&nbsp;', ' '),
+      ).click
```

Comment:
Just to clarify my understanding of this, this is because the Rails 7.1 defaults use a different (HTML5-capable) sanitizer, which doesn't swap the `&nbsp;` automatically like the previous sanitizer?

https://api.rubyonrails.org/classes/ActionView/Helpers/SanitizeHelper.html#method-i-strip_tags